hs_core/views/utils.py

```diff
@@ -46,7 +46,8 @@ def upload_from_irods(username, password, host, port, zone, irods_fnames, res_fi
         fname = os.path.basename(ifname.rstrip(os.sep))
         res_files.append(UploadedFile(file=tmpFile, name=fname, size=size))
 
-def authorize(request, res_id, edit=False, view=False, full=False, superuser=False, raises_exception=True):
+def authorize(request, res_id, discoverable=False, edit=False, view=False,
```

Comment:
@hyi If I only have permissions on a resource because it is discoverable, then I can only view the metadata, not the contents of the resource.  Therefore, I think we need to provide some way to distinguish between full view permissions and metadata view permissions.